R-package/tests/testthat/test_basic.R

```diff
@@ -189,6 +189,7 @@ test_that("lightgbm() rejects negative or 0 value passed to nrounds", {
         data = dtrain
         , params = params
         , nrounds = nround_value
+        , save_name = tempfile(fileext = ".model")
```

Comment:
This change looks unrelated to this PR. Can you please explain why this was necessary? If it's not related to this PR, please remove it.